quantum/q_fourier_transform.py

```diff
@@ -90,7 +103,6 @@ def quantum_fourier_transform(number_of_qubits: int = 3) -> qiskit.result.counts
 
 
 if __name__ == "__main__":
-    print(
-        f"Total count for quantum fourier transform state is: \
-    {quantum_fourier_transform(3)}"
-    )
```

Comment:
Why remove this?

Reply:
i got error relating to more no of lines.So i checked other there were no print statements..So but i can change it back